spack/s/singularity-legacy/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class SingularityLegacy(AutotoolsPackage):
+    """Singularity is a container platform focused on supporting 'Mobility of
+       Compute'. The software changed the installation method from AutoTools
+       to GoLang, so we have two separate package names to support that. The
+       legacy package is pre-version 3.0.0
+    """
+
+    homepage = "https://www.sylabs.io/singularity/"
+    url      = "https://github.com/sylabs/singularity/releases/download/2.5.2/singularity-2.5.2.tar.gz"
+    git      = "https://github.com/sylabs/singularity.git"
+
+    # Versions before 2.5.2 suffer from a serious security problem.
+    # https://nvd.nist.gov/vuln/detail/CVE-2018-12021
+    version('2.6-release', branch='vault/2.6-release', deprecated=True)
```

Comment:
Looks like this failed because the version is deprecated. Should we un-dreprecate it?

Reply:
oh hmm, let's just skip this one then, I always get nervous when I see "serious security problem" :P